exercises/shared/.docs/debug.md

```diff
@@ -0,0 +1,14 @@
+# Debugging
+
+To help with debugging, you can use the fact that all console output will be captured and shown in the test results window.
+You can output any value by passing it to `@show`.
+
+`@show` shows an expression and its result, while returning its result.
+
+```julia
+julia> @show uppercase("hello")
+uppercase("hello") = "HELLO"
+"HELLO"
+```
```

Comment:
I wonder if this should be `@show uppercase("hello");` to show what it prints rather than what it returns.

Reply:
I think a semicolon would be a bit confusing. How about a bigger example?

```julia
julia> function f(a, b)
           @show a b
           return (@show a^b) / 1000
       end
f (generic function with 1 method)

julia> f(3, 5)
a = 3
b = 5
a ^ b = 243
0.243

Reply:
````suggestion
```julia
@show uppercase("hello")
```
````

Reply:
> How about a bigger example?

Let's have both maybe? The bigger one assumes a bit more knowledge that might be too much earlier on in the track.

Reply:
````suggestion
```julia
@show uppercase("hello")
```

```julia
julia> function f(a, b)
           @show a b
           return (@show a^b) / 1000
       end
f (generic function with 1 method)

julia> f(3, 5)
a = 3
b = 5
a ^ b = 243
0.243
```
````